examples/README.rst

```diff
@@ -14,4 +14,5 @@ pySMT. Suggested order:
 8. efsmt.py: Shows how to combine two different solvers to solve an Exists Forall problem.
 9. allsmt.py: How to detect the logic of a formula and perform model enumeration.
 10. sudoku/sudoku.py: Solves sudoku problems on the command line using a simple encoding using either QF_LIA or QF_BV
-11. sudoku/gui.py: A small GUI that embeds sudoku.py and allows a user-friendly intercation with the sudoku solver.
+    sudoku/gui.py: A small GUI that embeds sudoku.py and allows a user-friendly intercation with the sudoku solver.
```

Comment:
I collapsed that into a single point, and added a README for the sudoku directory. The rationale is that the two examples do not show two aspects of pySMT, and thus should be considered as a single (although fairly complex) example.